graphql-server/Sources/App/GraphQL/GraphQL+ResponseGenerator.swift

```diff
@@ -0,0 +1,18 @@
+import GraphQL
+import HTTPTypes
+import Hummingbird
+
+extension GraphQLResult: ResponseGenerator {
```

Comment:
This generates the following warning
```
Extension declares a conformance of imported type 'GraphQLResult' to imported protocol 'ResponseGenerator'; this will not behave correctly if the owners of 'GraphQL' introduce this conformance in the future,
```
Can you remove the conformance from here and add it as follows below
```swift
#if hasFeature(RetroactiveAttribute)
extension GraphQLResult: @retroactive ResponseGenerator {}
#else
extension GraphQLResult: ResponseGenerator {}
#endif
```

Reply:
It looks like the same fix will need to be applied to the GraphQLError extension as well.